x/twap/logic_test.go

```diff
@@ -43,129 +43,254 @@ func newExpRecord(accum0, accum1 sdk.Dec) types.TwapRecord {
 	}
 }
 
+func (s *TestSuite) TestNewTwapRecord() {
```

Comment:
We should add a TODO for testing this with spot prices right?

(Not v12 blocking)